Update Google credentials section in README (#1795)
DataBiosphere/azul: README.md

### 2.3.2 Google credentials

> 1. Ask to be invited to the Google Cloud project. 

Reviewer: ```suggestion
1. Ask to be invited to a Google Cloud project. 
```

> For the HCA DCP v1 `dev`, `integration`, and `staging` deployments, this is `human-cell-atlas-travis-test`.

Reviewer: ```suggestion
For the lower HCA DCP v1 deployments (`dev`, `integration`, and `staging`), this would be `human-cell-atlas-travis-test`.
```

> 2. Log into `console.cloud.google.com` and select the desired project.

Reviewer: ```suggestion
2. Log into `console.cloud.google.com` and select that project.
```

   Google Cloud Platform -> Navigation menu -> IAM & admin
   ```

> 4. Create a service account for the project

Reviewer: ```suggestion
4. Create a service account for yourself in that project
```
